edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,79 @@
+Feature: patron tests
+
+  Background:
+    * url baseUrl
+    * callonce login testAdmin
+    * configure headers = {  'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+    * def uuid = call random_uuid
+    * def firstName = call random_string
+    * def lastName = call random_string
+    * def owner = random_string
+    * def amount = call random_numbers
+    * def status = 'Available'
+    * def requestId = call random_uuid
+    * def servicePointId = call random_uuid
+
+  Scenario: Return total fees/fines regardless of fee/fine being attached to an item for a patron.
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem') { itemId: 388830d5-95db-4528-95b6-4ec9d37d4086, materialTypeId: 388830d5-95db-4528-95b6-4ec9d37d4087,materialTypeName: TestMaterial, itemBarcode: 11111}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4056, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4086,materialTypeName: TestMaterial1, itemBarcode: 2222}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4089, itemId: 388830d5-95db-4528-95b6-4ec9d37d4056 }
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine'){ materialTypeId: 188830d5-95db-4528-95b6-4ec9d34d4086, itemId: 388830d5-95db-4528-95b6-4ec9d37d4086 }
```

Comment:
Please format these calls consistently, with single spaces between brackets, parameters etc 

(I cannot provide proper examples because github considers * as markdown in suggestions and so line continuation stops suggesting changes to some of these lines)